tests/scenarios/test_functionality_nimbus.py

```diff
@@ -37,3 +64,32 @@ def check_url_page_loads_correctly(selenium):
         EC.visibility_of_element_located((By.CSS_SELECTOR, ".c-navigation-logo-image")),
         message="Navigation logo not found",
     )
+
+
+@then("Firefox should still accept a copied string that is sent to the search bar")
+def copy_and_paste_string_to_url_bar(selenium, navigate_using_url_bar):
+    cmd_ctrl = Keys.COMMAND if sys.platform == "darwin" else Keys.CONTROL
+
+    selenium.get("http://www.allizom.org/en-US/")
+    el = selenium.find_element(
+        By.CSS_SELECTOR, "section.mzp-c-footer-section:nth-child(1) > h5:nth-child(1)"
+    )
+
+    # scroll down to text
+    selenium.execute_script("arguments[0].scrollIntoView(true);", el)
+
+    ActionChains(selenium).move_to_element(el).pause(1).double_click(el).key_down(
+        cmd_ctrl
+    ).send_keys("c").key_up(cmd_ctrl).perform()
+    navigate_using_url_bar(use_clipboard=True)
+
+    copied_text = el.text
+    WebDriverWait(selenium, 60).until(
+        EC.presence_of_element_located(
+            (By.CSS_SELECTOR, "body div#searchform div form div div div div div textarea")
```

Comment:
Not sure if this can be any more ... generic? (like an ID or classname to futureproof some potential DOM changes?)

Reply:
Yeah this one is hard, maybe there is a better way.

Reply:
I updated this to use the title, just in case google does some weird celebratory search page crap